barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.cpp

```diff
@@ -360,6 +363,59 @@ PairingPointAccumulatorIndices process_honk_recursion_constraints(
     return current_aggregation_object;
 }
 
+void process_ivc_recursion_constraints(MegaCircuitBuilder& builder,
```

Comment:
This logic is basically copy and pasted from its old home in `create_kernel_circuit`